docs/research/README.md

```diff
@@ -0,0 +1,3 @@
+# Dotgov Research
+
+This contains scripts for gathering data about dotgov usage and answering various secondary research questions.
```

Comment:
I have found "dotgov" means many different (sometimes diverging) things to different parties. I generally use the following for different flavors of .gov:
* DOTGOV - an act of Congress
* .gov - a TLD on the internet

GSA used to use DotGov/dotgov as shorthand for the program that exists to manage the .gov registry. Outside of hostnames ([home.dotgov.gov)](https://home.dotgov.gov), I intentionally don't use DotGov anymore because I think using it as a modifier of some _thing_ (".gov {program, registry, namespace}") is easier to intuit when reading and more technically accurate. 

Recognizing the repo is called "dotgov", when this sentence says "data about dotgov usage", is it talking about the new .gov registrar, the set (or some part) of domains and hosts that comprise the .gov namespace, or something else?